cmake/modules/AddSwift.cmake

```diff
@@ -674,26 +674,12 @@ endfunction()
 #   _add_swift_host_library_single(
 #     target
 #     name
-#     [MODULE_TARGETS]
 #     [SHARED]
 #     [STATIC]
 #     [SDK sdk]
 #     [ARCHITECTURE architecture]
-#     [DEPENDS dep1 ...]
-#     [LINK_LIBRARIES dep1 ...]
-#     [FRAMEWORK_DEPENDS dep1 ...]
-#     [FRAMEWORK_DEPENDS_WEAK dep1 ...]
 #     [LLVM_LINK_COMPONENTS comp1 ...]
-#     [C_COMPILE_FLAGS flag1...]
-#     [SWIFT_COMPILE_FLAGS flag1...]
-#     [LINK_FLAGS flag1...]
-#     [FILE_DEPENDS target1 ...]
```

Comment:
`add_swift_host_library` still defines `FILE_DEPENDS` as one of the parameters, but doesn't use it anywhere, so I'm wondering if it should be removed there as well.

Reply:
Hmm, perhaps I missed it?  It should be removed in the final view of things.  The individual steps are meant to be easier to digest.

Reply:
Hmm, it hasn't supported that for a while, it did however document it as supporting it before this set of changes.  Fake news! :-p